notify_listener.go

```diff
@@ -267,11 +275,16 @@ func (l *NotifyListener) processChangeset(event *store.Event) {
 		}
 
 		for k, v := range newValues {
-			// Maps are not supported. They can break checksum validation
-			// when re-marshaling. Pass the original JSON string instead.
-			switch v.(type) {
-			case map[string]interface{}:
-				v = string(newRawValues[k])
+			columnName := k
+			columnType, ok := columnTypes[fullTableName][columnName]
+			if !ok {
+				panic("column type could not be determined")
+			}
+
+			if columnType == "jsonb" || columnType == "json" {
```

Comment:
I know JSONB should be parsed differently, should regular JSON too?

Reply:
Going by the test case added. Both are fine.

Reply:
Both JSON and JSONB should be processed this way to avoid unmarshal to map[string]interface{}